src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Performance/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.ServiceBus.Performance
+{
+    class Program
+    {
+        private static readonly Stopwatch _stopwatch = Stopwatch.StartNew();
+
+        private static readonly byte[] _payload = new byte[1024];
+
+        private static long _messages;
+
+        static async Task Main(string[] args)
+        {
+            var maxInflight = (args.Length >= 1 ? int.Parse(args[0]) : 1);
+            Log($"Maximum inflight messages: {maxInflight}");
+
+            var messages = (args.Length >= 2 ? long.Parse(args[1]) : 10);
+
+            var connectionString = Environment.GetEnvironmentVariable("SERVICE_BUS_CONNECTION_STRING");
+            var entityPath = Environment.GetEnvironmentVariable("SERVICE_BUS_QUEUE_NAME");
```

Comment:
Should we consider taking the same approach for dynamic creation/teardown of a queue as the unit tests, rather than keeping one in residence?   The existing reference to the Service Bus project has a management surface baked in.

Reply:
This is a good suggestion.  One advantage of the test creating the queue is it can specify whatever parameters it thinks are best in terms of queue size, message retention, etc.  A current problem with the tests is you often need to manually delete and recreate the queue so it doesn't fill up.

What do you mean by "management surface baked in"?  Do you mean the .NET data-plane SDK has APIs to create queues?  Is this true for the data-plane SDK in all languages?  If so I am happy to add this feature.  I would also be fine creating the queue via the management-plane SDK, again assuming it's available in all languages, and assuming the setup is not too complicated (I believe the management-plane SDK might require a service principal).

Reply:
That's what I mean.  :)    (see:  [here](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/ServiceBus/data-plane/src/Microsoft.Azure.ServiceBus/Management).   So long as you've got a connection string that has `manage` rights to the Service Bus namespace, no additional principal is needed.

To my knowledge, Service Bus is unique in this regard; the functionality is a hold-over from the days before the management library officially existed.  IIRC, the client management area was deprecated at one point and feedback was loud enough that it was added back in.

Reply:
I believe the decision is to **not** add this functionality to the JS SDK, at least for now.  @ramya-rao-a, can you confirm or deny?

So, if we were going to add this to the perf tests, I'd want to do it in a consistent way across languages, which I believe would require using the managment plane libraries in each language and using a service principal.  It's borderline, but I agree this is probably better than manually creating the queue every time it gets full, so I'll investigate adding this to both tests in a later PR.